app/views/lit/localization_keys/_localizations_list.html.erb

```diff
@@ -30,26 +33,30 @@
         <div class="detail_wrapper">
           <table class="table table-bordered table-striped">
             <tr>
-              <th class="col-md-8">Translation</th>
-              <th class="col-md-2 text-center">Locale</th>
+              <th class="col-md-1"></th>
+              <th class="col-md-9">Translation</th>
+              <th class="col-md-1 text-center">Locale</th>
               <% unless lk.is_deleted? %>
-                <th class="col-md-2 text-center">Completed</th>
+                <th class="col-md-1 text-center">Completed</th>
               <% end %>
             </tr>
             <%- available_locales.each do |locale| %>
               <%- localization = localization_for(locale, lk) %>
               <tr>
+                <td>
+                  <% if localization %>
+                    <%= draw_icon 'clock-o', title: "Last updated at #{localization.updated_at.to_s(:db)}" %>
+                    <%= link_to lit.previous_versions_localization_key_localization_path(lk, localization, format: :js), class: "show_prev_versions #{'hidden' unless versions?(localization)}", remote: true do %>
```

Comment:
`show_prev_versions` -> `js-show-prev-versions` as it's not used for styling, just for targeting via jquery in one of js.erb actions